src/components/search/Search.jsx

```diff
@@ -10,45 +10,57 @@ import {
   fetchSinopiaSearchResults as fetchSinopiaSearchResultsCreator,
   fetchQASearchResults as fetchQASearchResultsCreator,
 } from 'actionCreators/search'
-import { clearSearchResults as clearSearchResultsAction } from 'actions/search'
+import { clearSearchResults as clearSearchResultsAction, setSearchResults } from 'actions/search'
 import SinopiaSearchResults from './SinopiaSearchResults'
 import QASearchResults from './QASearchResults'
 import SearchResultsPaging from './SearchResultsPaging'
 import SearchResultsMessage from './SearchResultsMessage'
 import Alert from '../Alert'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faSearch } from '@fortawesome/free-solid-svg-icons'
+import { faTrashAlt, faSearch } from '@fortawesome/free-solid-svg-icons'
 import searchConfig from '../../../static/searchConfig.json'
-import { selectSearchError, selectSearchUri, selectSearchOptions } from 'selectors/search'
+import {
+  selectSearchError, selectSearchQuery, selectSearchUri, selectSearchOptions,
+} from 'selectors/search'
 
 const Search = (props) => {
   const dispatch = useDispatch()
-  const fetchQASearchResults = (queryString, uri, startOfRange) => dispatch(fetchQASearchResultsCreator(queryString, uri, { ...searchOptions, startOfRange }))
```

Comment:
I rejiggered the order of this function to concentrate the `use*` functions towards the top.